feat(content): Gegno: Zydee event, Jyelio planet, and various tweaks

Saugia opened the pull request:
**Content (Artwork / Missions / Jobs)**

## Summary
This PR is meant to be a follow-up PR to #8750 to further establish the basis of future Gegno content. 

### Zydee

The Battle of Zydee occurs in Corroboration, and the player only sees it if they actually land on the planet during the appropriate timeframe. However, the battle happens regardless, and the planet sees its first occupants as, surprisingly, the ambitious Gegno Vi, who have invaded the system near the Scin's territory. This starts a sort of "see-saw" of resource-based invasion events that will occur throughout the story. The planet and system are now changed via an event, either within 1-3 days after you saw it, or after you have done the mission set before Aquiescence.

As a result, the Vesvii fleets are pulled back and lessened, while the fleets in Kneirc increase. This sets the tone of Gegno story moving forward, as this type of background system invasion will occur several times. Think of it like FW is indeed happening without the player, although it will still come (indirectly) as a result of the player engaging in missions.

In order for the system to properly appear inhabited on the map, it has to have initial trade prices regardless of it being uninhabited (see Wanderer campaign Korath systems). 

### Jyelio's Planet

While going through and planning out the backs-and-forths of Gegno map attacks (similar to FW), I've found that having a planet in Jyelio would be a useful and core inclusion to the important conflict near Vesvii. In the past I considered adding a landable within the Vesvii system itself for the Gegno to fight over, but I felt that was too direct for both them and the player. However, having one nearby works out, so I've added Uye Mpi to the Jyelio system. (Perhaps in the future I will make Vesvii itself a gas-giant landable.)

Similar to the above, it would need trade prices to work later on if it is occupied.

### Other

I've removed "Gegno" from two systems that have but don't need the government along with the attribute, changing them to just be Uninhabited. Kneirc also starts off as Uninhabited now, instead of "owned" by the Gegno Scin. Also gave any other systems trade prices that need them for future events.

I removed an unnecessary "accept".

I lowered the time for "quarg talks" a little so it occurs sooner.


## To Do
- [x] Uye Mpi description, landscape
- [x] Test events to make sure they work properly
- [x] Test fleet spawns to see if I feel comfortable with them
- [x] Balance trade prices (currently copypasta), see if any more future "occupant" systems need any

## Save File
This save file can be used to test these changes: 
[Hueim.Firlas.Gegno.Start~Hueim.Pre.Zydee.txt](https://github.com/user-attachments/files/16378956/Hueim.Firlas.Gegno.Start.Hueim.Pre.Zydee.txt)
Either land on Zydee nearby and trigger the event, waiting 1-3 days for it to change, or don't land there and do the rest of the Gegno content and watch Zydee change on the map.

roadrunner56 commented on data/map planets.txt:
```diff
@@ -4759,6 +4759,13 @@ planet "Ut Divitas"
+	attributes uninhabited
+	landscape land/bwerner6
+	description `Spanning lands of dust and bulky rock formations make up a majority of this world's details. However, unlike many other desert worlds, there are many basins of spring water and surprisingly calm weather patterns. The clear blue skies and tame wildlife commonly found on the planet give it a sense of serenity and beauty.`
+	description `	Nearby conflict between the warring Gegno factions often results in engagements within this system, though neither group has yet to commit to any large-scale foundation on the surface of this planet. A majority of skirmishes either occur in orbit above or follows the Gegno to neighboring systems.`
```
```suggestion
	description `	Nearby conflict between the warring Gegno factions often results in engagements within this system, though neither group has yet to commit to any large-scale foundation on the surface of this planet. A majority of skirmishes either occur in orbit above or follow the Gegno to neighboring systems.`
```

roadrunner56 commented on data/gegno/gegno events.txt:
```diff
@@ -29,6 +29,28 @@ event "gegno: quarg talks"
+	planet "Zydee"
+		attributes "gegno" "vi" "moon"
+		description `There is evidence that the nearby Gegno Scin have attempted to establish a presence on this barren moon in the form of several partially-constructed buildings and landing zones scattered about the surface. In recent times, however, the opposing Gegno Vi have fielded their own forces to take control of the system, and have begun a number of operations to convert existing infrastruct ...
+		spaceport `The Vi have made makeshift landing zones and outposts, such as this one, for the group of warriors they've distributed across the moon. Although their numbers are no more than than that of a small village, those who are present are hard-working and surprisingly have accomplished a lot during their time on the surface while the fleets above defend them.`
```
Personally I think that having both `hard-working and surprisingly have accomplished a lot` is a bit redundant. Up to you though.
```suggestion
		spaceport `The Vi have made makeshift landing zones and outposts, such as this one, for the group of warriors they've distributed across the moon. Although their numbers are no more than than that of a small village, those who are present have accomplished a considerable amount during their time on the surface while the fleets above defend them.`
```

Saugia commented:
Tested the events and they work properly. I've changed a bit of the numbers around and feel comfortable with them but if needed to change could. I also fixed something else in a different mission as an add-on to this PR. Otherwise it's about where I'd have it.

roadrunner56 commented on data/gegno/gegno events.txt:
```diff
@@ -29,6 +29,28 @@ event "gegno: quarg talks"
+		fleet "Scin Warfleet" 3000
+	planet "Zydee"
+		attributes "gegno" "vi" "moon"
+		description `There is evidence that the nearby Gegno Scin have attempted to establish a presence on this barren moon in the form of several partially-constructed buildings and landing zones scattered about the surface. In recent times, however, the opposing Gegno Vi have fielded their own forces to take control of the system, and have begun a number of operations to convert existing infrastruct ...
```
```suggestion
		description `There is evidence that the nearby Gegno Scin have attempted to establish a presence on this barren moon in the form of several partially-constructed buildings and landing zones scattered about the surface. In recent times, however, the opposing Gegno Vi have fielded their own forces to take control of the system, and have begun a number of operations to convert existing infrastructure and set up posts of their own, claiming the moon in their name.`
```

roadrunner56 reviewed (approved):
If you've playtested it and it works, I think this is good to go.
